### PR TITLE
Edge 127.0.2651.98-1 => 127.0.2651.105-1

### PR DESCRIPTION
**Pull request by uberhacker**
##
Tested & (not) Working properly:
- [x] `x86_64` Unable to launch in hatch m126 container
- [ ] `i686`
- [ ] `armv7l`
##
- [x] This PR has no manifest .filelist changes. _(Package changes have neither added nor removed files.)_
##
### Run the following to get this pull request's changes locally for testing.
```bash
CREW_REPO=https://github.com/uberhacker/chromebrew.git CREW_BRANCH=update-edge crew update \
&& yes | crew upgrade
```

### CODE REVIEW

File: .github/workflows/Unit-Test.yml [hunk 1]
```diff
@@ -28,9 +28,12 @@ jobs:
             echo "ALL_CHANGED FILES is/are ${ALL_CHANGED_FILES}." && \
             export CHANGED_PACKAGES="$(echo ${ALL_CHANGED_FILES} | sed -e 's,packages/,,g' -e 's,\.rb,,g' | sort)" && \
             echo "CHANGED PACKAGES is/are ${CHANGED_PACKAGES}." && \
-            export X86_64_PACKAGES="$(for p in ${CHANGED_PACKAGES}; do grep -q "compatibility.*all\|x86_64" packages/$p.rb && echo $p; done | xargs)" && \
+            export SAME_GLIBC_PACKAGES="$(for i in ${CHANGED_PACKAGES} ; do if ! grep -q min_glibc packages/$i.rb; then echo $i ; else grep min_glibc packages/$i.rb | tr -d \"'\" | awk '{exit $2 < 2.27}' || echo $i ; fi ; done | xargs)" && \
+            echo "PR #${PR_NUMBER} has these Glibc 2.27 compatible packages: ${SAME_GLIBC_PACKAGES}" && \
```
Comment (uberhacker) [inline .github/workflows/Unit-Test.yml:32]: 2.27 would be the lower glibc version, right?  If we set `min_glibc` in the package, it usually means a glibc version higher than 2.27 is needed.

File: .github/workflows/Unit-Test.yml [hunk 2]
```diff
@@ -28,9 +28,12 @@ jobs:
             echo "ALL_CHANGED FILES is/are ${ALL_CHANGED_FILES}." && \
             export CHANGED_PACKAGES="$(echo ${ALL_CHANGED_FILES} | sed -e 's,packages/,,g' -e 's,\.rb,,g' | sort)" && \
             echo "CHANGED PACKAGES is/are ${CHANGED_PACKAGES}." && \
-            export X86_64_PACKAGES="$(for p in ${CHANGED_PACKAGES}; do grep -q "compatibility.*all\|x86_64" packages/$p.rb && echo $p; done | xargs)" && \
+            export SAME_GLIBC_PACKAGES="$(for i in ${CHANGED_PACKAGES} ; do if ! grep -q min_glibc packages/$i.rb; then echo $i ; else grep min_glibc packages/$i.rb | tr -d \"'\" | awk '{exit $2 < 2.27}' || echo $i ; fi ; done | xargs)" && \
```
Comment (uberhacker) [inline .github/workflows/Unit-Test.yml:31]: `tr -d\'` should work.  No need to enclose with double quotes.

File: .github/workflows/Unit-Test.yml (hunk 1 above)
Comment (satmandu) [reply .github/workflows/Unit-Test.yml:32]: Awk's exit code is the reverse of most unix commands...

Comment (uberhacker) [reply .github/workflows/Unit-Test.yml:32]: That is fine.  I'm just mentioning the message should say `Glibc 2.27 incompatible` instead of compatible.

**Review by satmandu — APPROVED** [review]

**Comment by satmandu** [comment]
The unit tests are passing now...

**Review by uberhacker — COMMENTED** [review]
LGTM.

**Review by supechicken — APPROVED** [review]

**Pull request merged**